src/librustc_middle/mir/interpret/allocation.rs

```diff
@@ -565,7 +565,7 @@ impl<'tcx, Tag: Copy, Extra> Allocation<Tag, Extra> {
         })
     }
 
-    pub fn mark_definedness(&mut self, ptr: Pointer<Tag>, size: Size, new_state: bool) {
+    pub fn mark_init(&mut self, ptr: Pointer<Tag>, size: Size, new_state: bool) {
```

Comment:
`new_state` is not very clear; maybe `is_init` or so?

Reply:
The latter does make more sense.